library_generation/cli/entry_point.py

```diff
@@ -163,6 +163,15 @@ def __generate_repo_and_pr_description_impl(
     )
 
 
+def __needs_full_repo_generation(config_change: ConfigChange) -> bool:
```

Comment:
nit: private functions are [recommended](https://g3doc.corp.google.com/eng/doc/devguide/py/style/index.md?cl=head#316-naming) to have single underscores

Reply:
I changed the function name in this file.

We can gradually change functions in other files in future PRs.

Reply:
Thanks @diegomarquezp, TIL.